--- /home/amax/anaconda3/envs/mypytorch/lib/python3.6/site-packages/torch/nn/modules/activation.py
+++ /home/amax/anaconda3/envs/mypytorch/lib/python3.6/site-packages/torch/nn/modules/activation.py
@@ -1,15 +1,16 @@
 class Sigmoid(Module):
-    r"""Applies the Sigmoid function element-wise.
+    r"""Applies the element-wise function:
 
     .. math::
-        \text{Sigmoid}(x) = \sigma(x) = \frac{1}{1 + \exp(-x)}
+        \text{Sigmoid}(x) = \frac{1}{1 + \exp(-x)}
 
 
     Shape:
-        - Input: :math:`(*)`, where :math:`*` means any number of dimensions.
-        - Output: :math:`(*)`, same shape as the input.
+        - Input: :math:`(N, *)` where `*` means, any number of additional
+          dimensions
+        - Output: :math:`(N, *)`, same shape as the input
 
-    .. image:: ../scripts/activation_images/Sigmoid.png
+    .. image:: scripts/activation_images/Sigmoid.png
 
     Examples::
 
@@ -18,9 +19,7 @@
         >>> output = m(input)
     """
 
-    def forward(self, input: Tensor) -> Tensor:
-        """
-        Runs the forward pass.
-        """
+    @weak_script_method
+    def forward(self, input):
         return torch.sigmoid(input)
 